src/main/java/com/ciandt/techgallery/persistence/dao/impl/TechnologyRecommendationDAOImpl.java

```diff
@@ -84,4 +85,19 @@ public TechnologyRecommendation findByComment(TechnologyComment comment) {
     }
   }
 
+  @Override
+  public String findAllRecommendationsIdsStartingFrom(Date date) {
+    final Objectify objectify = OfyService.ofy();
+    final List<TechnologyRecommendation> allRecommendations =
+        objectify.load().type(TechnologyRecommendation.class).list();
```

Comment:
The problem here is that to obtain the timestamp from a recommendation we have to get its comment and get its timestamp then compare it. Maybe if we had another timestamp inside recommendation is a better practice.